openprescribing/media/css/general.less

```diff
@@ -16,7 +16,7 @@ h1, h2, h3, h4, h5, h6, .navbar-brand, #navbar {
 h1, h2, h3, h4, h5, h6 {
   text-shadow: 1px 1px 1px rgba(150, 150, 150, 0.64);
 }
-.container p {
+.container p, .alert {
```

Comment:
this will affect all alerts not just the new one, but I assume that looks sensible in other places too?